R/step_lag_difference.R

```diff
@@ -15,14 +15,14 @@
 #' @family row operation steps
 #' @export
 #' @examples
-#' r <- epi_recipe(case_death_rate_subset) %>%
+#' r <- epi_recipe(covid_case_death_rates) %>%
 #'   step_lag_difference(case_rate, death_rate, horizon = c(7, 14)) %>%
 #'   step_epi_naomit()
 #' r
 #'
 #' r %>%
-#'   prep(case_death_rate_subset) %>%
-#'   bake(case_death_rate_subset)
+#'   prep(covid_case_death_rates) %>%
+#'   bake(covid_case_death_rates)
```

Comment:
```suggestion
#'   bake(new_data = NULL)
```

(unrelated to the PR, but may as well while we're here.